WordPress/Classes/ViewRelated/Reader/ReaderCardDiscoverAttributionView.swift

```diff
@@ -43,7 +43,7 @@ import WordPressShared
         if contentProvider?.sourceAttributionStyle() == SourceAttributionStyle.Post {
             configurePostAttribution(contentProvider!)
         } else if contentProvider?.sourceAttributionStyle() == SourceAttributionStyle.Site {
-            configureSiteAttribution(contentProvider!)
+            configureSiteAttribution(contentProvider!, verboseAttribution: false)
```

Comment:
(Sorry about being nipticky!) how can we verify this one? (perhaps it should live in another PR?)